src/main/java/stsjorbsmod/powers/DamnationPower.java

```diff
@@ -0,0 +1,45 @@
+package stsjorbsmod.powers;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.NonStackablePower;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import stsjorbsmod.util.CardMetaUtils;
+
+public class DamnationPower extends CustomJorbsModPower implements NonStackablePower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(DamnationPower.class);
+    public static final String POWER_ID = STATIC.ID;
+    private boolean isAutoUpgrade;
+
+    public DamnationPower(final AbstractCreature owner, boolean isAutoUpgrade) {
+        super(STATIC);
+
+        this.owner = owner;
+        this.isAutoUpgrade = isAutoUpgrade;
+
+        updateDescription();
+    }
+
+    @Override
+    public void onUseCard(AbstractCard card, UseCardAction action) {
```

Comment:
Should we exclude cards that aren't in the master deck? If not, should we at least exclude transient copies of cards used by "play next card twice" type mechanics? (Probably need to ask discord)

Reply:
Still asking about cards that aren't in the master deck

Reply:
cards not in the master deck will consume the charge without transforming